core/workspace_comment/workspace_comment.js

```diff
@@ -30,11 +30,13 @@ goog.provide('Blockly.WorkspaceComment');
  * Class for a workspace comment.
  * @param {!Blockly.Workspace} workspace The block's workspace.
  * @param {string} content The content of this workspace comment.
+ * @param {number} height Height of the comment.
+ * @param {number} width Width of the comment.
  * @param {string=} opt_id Optional ID.  Use this ID if provided, otherwise
  *     create a new ID.
  * @constructor
  */
-Blockly.WorkspaceComment = function(workspace, content, opt_id) {
+Blockly.WorkspaceComment = function(workspace, content, height, width, opt_id) {
```

Comment:
height and width need defaults since this is an API change. It's fine to declare them as required, though.

Reply:
Not really an API change since we've never merged this code before *and* it wasn't working the old way.